modules/log-collector-resources-scheduling.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/log-forwarding.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log-collector-resources-scheduling_{context}"]
+= Configuring resources and scheduling for logging collectors
+
+Administrators can modify the resources or scheduling of the collector by creating a `ClusterLogging` custom resource (CR) that is in the same namespace and has the same name as the `ClusterLogForwarder` CR that it supports.
+
+The applicable stanzas for the `ClusterLogging` CR when using multiple log forwarders in a deployment are `managmentState` and `collection`. All other stanzas are ignored.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {clo} version 5.8 or newer.
+* You have created a `ClusterLogForwarder` CR.
+
+.Procedure
+
+. Create a `ClusterLogging` CR that supports your existing `ClusterLogForwarder` CR:
++
+.Example `ClusterLogging` CR YAML
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name:  <name> <1>
```

Comment:
For the YAML callouts, [comment](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-asciidoc-callouts-in-the-yaml) them out as below:
```suggestion
  name:  <name> # <1>
```

Reply:
Good call, I had forgotten about this!

Reply:
Honestly hoping this round of squad duty finally cements it in my head for my own work :upside_down_face: